flake8_quotes/__init__.py

```diff
@@ -9,34 +9,60 @@ class QuoteChecker(object):
     name = __name__
     version = __version__
 
-    QUOTES = {
+    INLINE_QUOTES = {
         # When user wants only single quotes
         '\'': {
-            'good_single': '\'',
-            'good_multiline': '\'\'\'',
-            'bad_single': '"',
-            'bad_multiline': '"""',
+            'good': '\'',
+            'bad': '"',
         },
+
+        # When user wants only double quotes
         '"': {
-            'good_single': '"',
-            'good_multiline': '"""',
-            'bad_single': '\'',
-            'bad_multiline': '\'\'\'',
+            'good': '"',
+            'bad': '\'',
+        },
+    }
+
+    MULTILINE_QUOTES = {
+        # When user wants only single multiline quotes
+        '\'': {
+            'good': '\'\'\'',
+            'bad': '"""'
         },
+
+        # When user wants only double multiline quotes
+        '"': {
+            'good': '"""',
+            'bad': '\'\'\''
+        }
     }
 
+    MULTILINE_QUOTES_STRINGS = ('\'\'\'', '"""')
+
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
     @classmethod
     def add_options(cls, parser):
-        parser.add_option('--quotes', default='\'', action='store',
+        parser.add_option('--quotes', default='\'', action='store', dest='inline_quotes',
                           help='Quote to expect in all files (default: \')')
-        parser.config_options.append('quotes')
+        parser.add_option('--multiline-quotes', action='store',
+                          help='Multiline quote to expect in all files (disabled by default)')
+        parser.config_options.extend(['quotes', 'multiline_quotes'])
 
     @classmethod
     def parse_options(cls, options):
-        cls.quotes = cls.QUOTES[options.quotes]
+        if hasattr(options, 'multiline_quotes'):
+            multiline_quotes = options.multiline_quotes
+        else:
+            multiline_quotes = options.inline_quotes
+
+        cls.inline_quotes = cls.INLINE_QUOTES[options.inline_quotes]
+
+        if multiline_quotes is not None:
+            cls.multiline_quotes = cls.MULTILINE_QUOTES[multiline_quotes]
+        else:
+            cls.multiline_quotes = None
```

Comment:
I think this logic is currently too spread out for multiline quotes. It should look something like:

```python
def parse_options(cls, options):
  if hasattr(options, 'multiline_quotes'):
    cls.multiline_quotes = cls.MULTILINE_QUOTES[options.multiline_quotes]
  else:
    cls.multiline_quotes = None

  cls.inline_quotes = cls.INLINE_QUOTES[options.inline_quotes]
```